src/datafile.rs

```diff
@@ -49,7 +49,12 @@ fn write(file: path::PathBuf, data: Vec<u8>) -> Result<(), io::Error> {
 }
 
 fn rpg_dir() -> path::PathBuf {
-    dirs::home_dir().unwrap().join(".rpg")
+    let home_dir = dirs::home_dir().unwrap().join(".rpg");
```

Comment:
Can you leave a comment explaining that home is checked first because that was the default in previous versions?

Reply:
This is only when the folder does not exist. We could refactor the code and make it more compact.
The reason we don't just put dirs::data_dir here is because we don't want people to lose their data after updating the program.

Reply:
Okay done